diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+0, 5
```

Comment:
result is equal to 5.  When calling strangeAdd(0, -5), the first conditional expression evaluates to false since 0 > 0 is false and -5 > 0 are both false.  The line "return a - b;" then runs, returning 0 - (-5), which equals 5.